sig/steep/server/custom_methods.rbs

```diff
@@ -124,6 +124,18 @@ module Steep
 
         def self.response: (String id, result) -> untyped
       end
+
+      module Refork
```

Comment:
Can you add a comment to explain what the request is doing, who sends the request, ...?